### PR TITLE
Condition reason/message cleanup and CLI display update

### PR DESCRIPTION
**Pull request by jc-rh**

### CODE REVIEW

**Comment by fontivan** [comment]
@jc-rh do you have an example of the output of `oc get cgu` with these changes?

**Comment by jc-rh** [comment]
> @jc-rh do you have an example of the output of `oc get cgu` with these changes?

Right, forgot to update the kuttl assert yamls.

File: README.md [hunk 1]
```diff
@@ -38,32 +38,30 @@ A full table of the conditions and their appropriate reasons and types are:
...
-  `BackupSucceeded` | True | Completed | Backup is completed for all clusters|
+  `BackupSucceeded` | True | BackupCompleted | Backup is completed for all clusters|
   | | True | PartiallyDone | Backup failed for x clusters |
-   | | False | NotEnabled | Backup will start when enable is true |
   | | False | InProgress | Backup is in progress for x clusters|
   | | False | Failed | Backup failed for all the clusters |
-  `Progressing`| True | InProgress| The ClusterGroupUpgrade CR has upgrade policies that are still non compliant|
-  | | False | Completed | The ClusterGroupUpgrade CR has all clusters compliant with all the managed policies |
+  `Progressing`| True | InProgress| Remediating non-compliant policies|
+  | | False | Completed | All clusters compliant with all the managed policies |
```
Comment (sabbir-47) [inline README.md:58]: nit. All clusters `are` compliant with all the managed policies

**Pull request merged**